src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -2645,6 +2652,12 @@ private void EmitSizeOfExpression(BoundSizeOfOperator boundSizeOfOperator)
             EmitSymbolToken(type, boundSizeOfOperator.SourceType.Syntax);
         }
 
+        private void EmitMethodTokenExpression(BoundMethodToken node)
+        {
+            _builder.EmitOpCode(ILOpCode.Ldtoken);
+            EmitSymbolToken(node.Method, node.Syntax, null, encodeAsRawToken: true);
```

Comment:
Is this supposed to be done only for definitions? Consider asserting that the method is definition then. It would be good to assert the result type of the node, unless there is a better place to do this.